### PR TITLE
FT21-1-validation to QC

### PR DESCRIPTION
**Pull request by amilan17**

### CODE REVIEW

**Review by amilan17 — COMMENTED** [review]
added all changes to doc....

File: tables_en/11-03.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
 notation,name,description
-beginning,Beginning,Time stamps indicate the end of a period covering the range up to but excluding the preceding time stamp
-end,End,Time stamps indicate the middle of a period beginning at the middle of the range described by this and the preceding time stamp and ending right before the middle of the range described by this and the following time stamp.
+beginning,Beginning,Time stamps indicate the beginning of a period covering the range up to but excluding the subsequent time stamp.
+end,End,Time stamps indicate the end of a period covering the range back to but excluding the preceding time stamp.
 inapplicable,inapplicable,None of the codes in the table are applicable in the context of this particular observation.
```
Comment (chenxiaoxia2019) [inline tables_en/11-03.csv:4]: @amilan17 https://github.com/wmo-im/wmds/issues/197 This issue mentions that deprecate inapplicable in 2-02.csv. just double check that inapplicable should be deleted here or not.

File: tables_en/superseded.txt [hunk 2]
```diff
@@ -5,4 +5,15 @@ https://codes.wmo.int/wmdr/ObservingMethodAtmosphere/262,https://codes.wmo.int/w
...
+https://codes.wmo.int/wmdr/ProgramAffiliation/RBONpilot,https://codes.wmo.int/wmdr/ProgramAffiliation/RBON
+https://codes.wmo.int/wmdr/ProgramAffiliation/RBSN,https://codes.wmo.int/wmdr/ProgramAffiliation/RBON
+https://codes.wmo.int/wmdr/ProgramAffiliation/RBSNp,https://codes.wmo.int/wmdr/ProgramAffiliation/RBON
+https://codes.wmo.int/wmdr/ProgramAffiliation/RBSNs,https://codes.wmo.int/wmdr/ProgramAffiliation/RBON
+https://codes.wmo.int/wmdr/ProgramAffiliation/RBSNsp,https://codes.wmo.int/wmdr/ProgramAffiliation/RBON
+https://codes.wmo.int/wmdr/ProgramAffiliation/RBSNst,https://codes.wmo.int/wmdr/ProgramAffiliation/RBON
+https://codes.wmo.int/wmdr/ProgramAffiliation/RBSNt,https://codes.wmo.int/wmdr/ProgramAffiliation/RBON
 https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_610,https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_12013
+https://codes.wmo.int/wmdr/ProgramAffiliation/IDAF,https://codes.wmo.int/wmdr/ProgramAffiliation/INDAAF
+https://codes.wmo.int/wmdr/ProgramAffiliation/ESRLCCG,https://codes.wmo.int/wmdr/ProgramAffiliation/NOAAGML
```
Comment (chenxiaoxia2019) [inline tables_en/superseded.txt:18]: @amilan17 Since the ESRLCCG is superseded, do we need the link here?

**Review by chenxiaoxia2019 — COMMENTED** [review]
@amilan17 All issues have been reviewed.

File: tables_en/superseded.txt (hunk 2 above)
Comment (amilan17) [reply tables_en/superseded.txt:18]: yes. this is the table of superseded values.

**Pull request merged**